www/javascripts/missions.js

```diff
@@ -80,13 +100,7 @@
 					playerId: '=',
 					missionId: '='
 				},
-				controller: ['Errors', 'Players', function (Errors, Players) {
-					$playerMissionCtrl = this;
-					Players.getMissionStatus(this.playerId, this.missionId,
-						function(data) {
-							$playerMissionCtrl.missionStatus = data;
-						}, Errors.handleError);
-				}],
+				controller: 'PlayerMissionCtrl',
```

Comment:
I factorized the controller declaration so it can be more easily reused and simply passed a handle as to which controller to use on the creation of this directive.